.eslintrc.json

```diff
@@ -4,43 +4,70 @@
   ],
   "ignorePatterns": [
     "docs",
+    "tools",
     "dist",
-    "test/files/*"
+    "website.js",
+    "test/files/*",
+    "benchmarks"
```

Comment:
Do we really want to avoid linting benchmarks and `website.js`? Doesn't seem like the right thing to do, but I might be missing something